.github/workflows/dismiss-approvals.yml

```
@@ -14,23 +14,37 @@ jobs:
-                    --jq '[.commits[] | select(.parents|length==1)] | length')
-        echo "nonmerge=$NON_MERGE" >>"$GITHUB_OUTPUT"
+      - name: Compute PR diff hash
+        id: diff
+        run: |
+          PR=${{ github.event.pull_request.number }}
+          # Hash current PR diff (files + patches)
+          HASH=$(gh api repos/${{ github.repository }}/pulls/$PR/files --paginate \
+            --jq '.[] | [ .filename, .status, (.patch // "") ] | join("\u0001")' \
+            | sha256sum | cut -d" " -f1)
+          echo "hash=$HASH" >> "$GITHUB_OUTPUT"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)